src/main/java/com/github/jamesnetherton/zulip/client/api/message/Message.java

```diff
@@ -58,6 +58,9 @@ public class Message {
     private String senderRealm;
 
     private String stream;
+    
+    @JsonProperty("stream_id")
```

Comment:
Jackson will automatically serialize the JSON field names to snake case. So you can simplify to:
```suggestion
    @JsonProperty
```

Reply:
I wanted to user lombok @Data annotation and others. do you think it's a good idea

Reply:
I'm not a big fan of Lombok unfortunately.

I know it means potentially writing more boilerplate, but I don't mind taking that hit or getting my IDE / chatbots to generate code for me if needed.